src/olympia/devhub/templates/devhub/agreement.html

```diff
@@ -15,11 +15,7 @@
     {{ _('I have read and accept this Agreement and the Rules and Policies') }}.
   </p>
 
-  {% if agreement_form.recapcha %}
-    <p>
-      {{ agreement_form.recaptcha }}
-    </p>
-  {% endif %}
+  {{ agreement_form.recaptcha }}
```

Comment:
The field is being deleted from the form in case the waffle isn't active so it's safe to use like that.